cmd/maddy-pam-helper/main.c

```diff
@@ -0,0 +1,86 @@
+#define _POSIX_C_SOURCE 200809L
+#include <stdio.h>
+#include <stdlib.h>
+#include <security/pam_appl.h>
+
+/*
+I really doubt it is a good idea to bring Go to the binary whose primary task
+is to call libpam using CGo anyway.
+*/
+
+static struct pam_response *reply;
+
+static int conv_func(int num_msg, const struct pam_message **msg, struct pam_response **resp, void *appdata_ptr) {
+    // Silence "unused parameter" warnings.
```

Comment:
Can you add `-Wno-unused-parameter` to CFLAGS so that we can remove this and silence warnings?